packages/xstate-inspect/src/types.ts

```diff
@@ -60,7 +61,7 @@ export type ParsedReceiverEvent =
   | {
       type: 'service.register';
       machine: AnyStateMachine;
-      state: AnyState;
+      state: StateConfig<any, any>;
```

Comment:
Note that this becomes incorrect with this change because it can be just any `TInternalState`